content/pages/comparisons/gitbook.html

```diff
@@ -0,0 +1,185 @@
+          <div class="ui padded basic segment">
+            <div class="ui relaxed list">
+
+              {%- set header_1 = "Create an account" %}
+              {%- set icon_1 = "fa-square-1" -%}
+              <span class="ui small teal header item">
+                <i class="fad {{ icon_1 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_1 }}
+                  <span class="sub header">Sign up with GitHub or your email.</span>
+                </span>
+              </span>
+
+              {%- set header_2 = "Import your project" %}
+              {%- set icon_2 = "fa-square-2" -%}
+              <span class="ui small header item">
+                <i class="fad {{ icon_2 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_2 }}
+                  <span class="sub header">Select your existing Git repositories with a 1-click interface.</span>
+                </span>
+              </span>
+
+              {%- set header_3 = "Add YAML config" %}
+              {%- set icon_3 = "fa-square-3" -%}
+              <span class="ui small header item">
+                <i class="fad {{ icon_3 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_3 }}
+                  <span class="sub header">Save this file as <code>.readthedocs.yaml</code>, and update the commands as necessary</span>
+                </span>
+              </span>
+
+              {%- set header_4 = "Your docs build on every commit" %}
+              {%- set icon_4 = "fa-square-4" -%}
+              <span class="ui small header item">
+                <i class="fad {{ icon_4 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_4 }}
+                  <span class="sub header">Like magic.</span>
+                </span>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  {% block homepage_bottom_callout %}
+  <section>
+    <div class="ui very padded container">
+      <div class="ui grid center aligned">
+        <div class="row">
+          <div class="column twelve wide computer sixteen wide tablet">
+            <div class="ui basic vertical huge segment">
+              <h2 class="ui center aligned header">
+                Still not convinced? Try it yourself  <i class="fad fa-rocket icon"></i>
+              </h2>
+
+              <p>
+                GitBook can be useful only if you are in-love with <strong>their own non open source software solution</strong>.
+                However, for publishing <em>good documentation</em> without vendor lock-in and plenty of features that your readers will want,
+                Read the Docs is the all-in-one solution that gets you started with all those features from the beginning.
+              </p>
```

Comment:
```suggestion
              <p>
                GitBook is a great product, but gives few options because of their <strong>their closed source software solution</strong>.
                For publishing <em>great documentation</em> without vendor lock in and features that your readers will love,
                Read the Docs is the all-in-one solution that provides everything you need out of the box, from analytics to SSO.
              </p>
```

Reply:
```suggestion
              <p>
                GitBook is a great product, but gives few options because of <strong>their closed source software solution</strong>.
                For publishing <em>great documentation</em> without vendor lock in and features that your readers will love,
                Read the Docs is the all-in-one solution that provides everything you need out of the box, from analytics to SSO.
              </p>
```

Remove duplicated _their_.